### PR TITLE
4/21

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2023-04-21.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+[![go-mysql-server mascot](../images/go-mysql-server-small.png)](https://github.com/dolthub/go-mysql-server)
+
+We hope you like it. We had some debate about bowtie on or off. I made the case that it makes him look like he's leading a double life as [Magic `go-mysql-server`](https://www.imdb.com/title/tt1915581/). Reply to this email for more details.
+
+### Clone a Hosted Dolt
+
+You can now [clone a Hosted Dolt](https://www.dolthub.com/blog/2023-04-17-cloning-a-hosted-database/)! Problem in production? Clone a copy to your desktop and start debugging as if it were production. 
+
+[Hosted Dolt](https://hosted.doltdb.com/) is the best way to run Dolt in production. You can [clone it](https://www.dolthub.com/blog/2023-04-17-cloning-a-hosted-database/) and get insights or make changes on a branch using a [versioned SQL workbench](https://www.dolthub.com/blog/2023-03-17-dolthub-vs-hosted-workbench/). This comes with all the convenience and enterprise support you expect from a cloud hosted database. You can run a hosted Dolt on AWS on Google Cloud.
```
Comment (bpf120) [inline emails/2023-04-21.md:13]:
```suggestion
[Hosted Dolt](https://hosted.doltdb.com/) is the best way to run Dolt in production. You can [clone it](https://www.dolthub.com/blog/2023-04-17-cloning-a-hosted-database/) and get insights or make changes on a branch using a [versioned SQL workbench](https://www.dolthub.com/blog/2023-03-17-dolthub-vs-hosted-workbench/). This comes with all the convenience and enterprise support you expect from a cloud hosted database. You can run a hosted Dolt on AWS or Google Cloud.
```

File: emails/2023-04-21.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+### Clone a Hosted Dolt
+
+You can now [clone a Hosted Dolt](https://www.dolthub.com/blog/2023-04-17-cloning-a-hosted-database/)! Problem in production? Clone a copy to your desktop and start debugging as if it were production. 
+
+[Hosted Dolt](https://hosted.doltdb.com/) is the best way to run Dolt in production. You can [clone it](https://www.dolthub.com/blog/2023-04-17-cloning-a-hosted-database/) and get insights or make changes on a branch using a [versioned SQL workbench](https://www.dolthub.com/blog/2023-03-17-dolthub-vs-hosted-workbench/). This comes with all the convenience and enterprise support you expect from a cloud hosted database. You can run a hosted Dolt on AWS or Google Cloud.
+
+### Dolt Architecture Primer
+
+I've long been ashamed of our lack of a good architecture introduction in [our documentation](https://docs.dolthub.com/introduction/what-is-dolt). That is [now fixed](https://docs.dolthub.com/architecture/architecture). This [blog](https://www.dolthub.com/blog/2023-04-19-dolt-architecture-intro/) explains why we built Dolt the way we did and the building blocks we used.
```
[the remark this answers is not in the document]
Comment (timsehn) [reply emails/2023-04-21.md:17]: What do you mean?

[the remark this answers is not in the document]
Comment (timsehn) [reply emails/2023-04-21.md:17]: Like 

Questions like "Does Dolt contain any MySQL or Git Code?" and "Why is Dolt written in Golang?" are answered once and for all.

[the remark this answers is not in the document]
Comment (bpf120) [reply emails/2023-04-21.md:17]: people always seem curious about why Dolt is MySQL syntax and written in Go. The blog answers that. 'If you're curious why we choose MySQL or wrote Dolt in Go, this blog is for you.'

**Comment by bpf120** [comment]
LGTM

**Pull request merged**